### PR TITLE
Alexa Belsky

### PR DESCRIPTION
**Pull request by kevkcc**
Failing test is due to space camp

### CODE REVIEW

File: src/clj/game/cards/assets.clj [hunk 1]
```diff
@@ -88,6 +88,35 @@
...
+   "Alexa Belsky"
+   {:abilities [{:label "[Trash]: Shuffle all cards in HQ into R&D"
+                 :effect (req (trash state side card)
+                              (if (pos? (:credit runner))
+                                (do (show-wait-prompt state :corp "Runner to decide whether or not to prevent Alexa Belsky")
+                                    (resolve-ability
+                                      state side
+                                      {:prompt "How many credits?"
+                                       :choices :credit :player :runner
+                                       :msg (msg "shuffle " (- (count (:hand corp)) (int (/ target 2))) " card"
```
Comment (Saintis) [inline src/clj/game/cards/assets.clj]: You can use the `qout` function to get integer division.

File: src/clj/game/cards/assets.clj [hunk 2]
```diff
@@ -88,6 +88,35 @@
                                                :msg (msg "trash " (join ", " (map :title targets)))))]
                              (continue-ability state side ab agg nil)))})
 
+   "Alexa Belsky"
+   {:abilities [{:label "[Trash]: Shuffle all cards in HQ into R&D"
+                 :effect (req (trash state side card)
+                              (if (pos? (:credit runner))
```
Comment (Saintis) [inline src/clj/game/cards/assets.clj]: The runner could use credits on Net Mercur for this, hence could prevent cards even on 0 credits.

**Comment by kevkcc** [comment]
Good catches!

**Comment by mtgred** [comment]
👍

**Pull request merged**